contracts/test/EVMCompatabilityTester.js

```diff
@@ -428,7 +428,7 @@ describe("EVM Test", function () {
         });
 
         describe("Differing maxPriorityFeePerGas and maxFeePerGas", async function() {
-          testCases.forEach(async ([name, maxPriorityFeePerGas, maxFeePerGas]) => {
+          for (const [name, maxPriorityFeePerGas, maxFeePerGas] of testCases) {
```

Comment:
what's the difference?

Reply:
Not much, but my ide linter complains that the async function promise wasn't being used, and gave me yellow lines, so I accepted its change suggestion.